### ISSUE TITLE
register the package

### ISSUE THREAD

**Issue by tbeason** [ericqu/LinearRegression.jl#39]
Nice work here, thanks for the effort.

At this point, the package is definitely mature enough to register in the general registry. Without registering, you create additional unnecessary complexities for anyone wanting to use the package. For example, Julia's built-in and Pluto's custom package management systems both now have fancy auto-management features that only work for registered packages.

**Comment by ericqu** [comment]
Thank you for the kind words.

That's a good point; I will register the package and update this issue once it is done.

**Comment by ericqu** [comment]
@tbeason, unfortunately, the name is already used (https://github.com/st--/LinearRegression.jl). 
Would you happen to have another name in mind? I thought of making a poll on the Julia Discourse to ask/find a new name.

**Comment by tbeason** [comment]
Yea I wondered about the name and if there might be a conflict.

LinearRegressionKit.jl?

**Comment by ericqu** [comment]
I added your suggestion to the poll, I will let it continue for a few days and then will change the name for the next release. 
I hope you are not too much in a rush.

**Comment by ericqu** [comment]
to keep you updated:
- the new name is LinearRegressionKit.jl, as you suggested. 
- the new repository is https://github.com/ericqu/LinearRegressionKit.jl. I will keep this repository and place a message in the Readme once the new repository is fully operational.
- I updated the content to the new name and modified the project file to satisfy the registration service. Tests are passing, and extra scripts are working okay.
- The registration status is available here: https://github.com/JuliaRegistries/General/pull/50138; it indicates a mandatory three-day waiting period.

If you think something is off, just let me know.

**Comment by ericqu** [comment]
The package LinearRegressionKit.jl is now registered; you can add the project using "add LinearRegressionKit" from the package manager prompt. Please open another issue if it doesn't work for you.
